Containerfile

```diff
@@ -15,11 +13,14 @@ RUN go mod download
 # build
 COPY . .
 
-ARG GOARCH
-RUN GOOS=linux GOARCH=$GOARCH CGO_ENABLED=0 go build -tags build -o /usr/local/bin/db-operator -ldflags="-X 'github.com/db-operator/db-operator/internal/helpers/common.OperatorVersion=${OPERATOR_VERSION}'" cmd/main.go
+RUN uname -a
+
+ARG TARGETARCH
+RUN GOOS=linux GOARCH=$TARGETARCH CGO_ENABLED=0 go build -tags build -o /usr/local/bin/db-operator cmd/main.go
 
 FROM registry.hub.docker.com/library/alpine:3.18
-LABEL org.opencontainers.image.authors="Nikolai Rodionov<allanger@zohomail.com>"
+RUN uname -a
```

Comment:
(minor) ... should not be needed in the final image layers, for debugging the one on line 16 should be good enough, or?